daemon/api_snaps.go

```diff
@@ -203,6 +203,7 @@ type snapInstruction struct {
 	Snaps                  []string               `json:"snaps"`
```

Comment:
also the sideload code  (see corresponding file) supproting `snap install foo.snap` should also support this, there we have a multpart form

Reply:
this doesn't seem addressed yet

Reply:
Yeah I missed this one, thanks for reminding me. Support for this is added in the latest commit